src/main/js/config/RepositoryConfig.tsx

```diff
@@ -44,6 +45,35 @@ type Props = {
   link: string;
 };
 
+export const TriggerButton: FC<{ link: string }> = ({ link }) => {
+  const [t] = useTranslation("plugins");
+  const [error, setError] = useState<Error | undefined>();
+  const [loading, setLoading] = useState<boolean>();
```

Comment:
`error` and `loading` sound like these are global states (eg. that they represent errors or loading state for the whole config view). This is not the case. I'd name them `triggerError` and `triggerLoading` or something like this to make this clear.